plugins/outputs/sql/README.md

```diff
@@ -112,6 +112,13 @@ through the convert settings.
   #  defaultvalue         = "TEXT"
   #  unsigned             = "UNSIGNED"
   #  bool                 = "BOOL"
+
+  ## This setting controls the behavior of the unsigned value. By default the
+  ## setting will take the integer value and append "unsigned" to it. The other
+  ## option is "literal", which will use the actual value the user provides to
```

Comment:
```suggestion
  ## setting will take the integer value and append the unsigned value to it. The other
  ## option is "literal", which will use the actual value the user provides to
```

The text here makes it sound like it appends the string "unsigned" but it appends the value of the unsigned setting.

Reply:
good catch